<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BotSetup</class>
 <widget class="QDialog" name="BotSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection wizard | Bot Account</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>CobraLink Token:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLineEdit" name="botToken"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLineEdit" name="streamerToken"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Streamer Token:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="generateButton">
     <property name="text">
      <string>Generate Tokens</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>save</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLineEdit" name="cobraToken"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Bot Token:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="errorLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
